src/main/entities/PasswordGenerator.java

```diff
@@ -1,14 +1,15 @@
-package src.passwordgenerator;
+package entities;
 
-import src.UI.DialogBox;
+import UI.DialogBox;
 
 public class PasswordGenerator {
 
     PasswordGenerator() {
+        //TODO: Move this to UIController
         DialogBox systemDialog = new DialogBox();
     }
     public static void main(String[] args) {
```

Comment:
Should this be removed?

Reply:
Yes, this would be removed by Rutvik in his PR.